apps/frontend/src/pages/auth/libs/components/sign-in-form/sign-in-form.tsx

```diff
@@ -1,3 +1,5 @@
+import { UserValidationRule } from "shared/src/modules/users/libs/enums/enums.js";
```

Comment:
It should be exported from
```
import { UserValidationRule } from "~/modules/users/users.js";
```
in `~/modules/users/users.js` from: 
```
"./libs/enums/enums.js"
```
and in enums from:
```
import { UserValidationRule } from "shared";
```
Everything in shared follows a long path to index.ts (please take a look at other exports in shared).

Maybe it would be better to just create separate constant in frontend user module and store NO_ERROR that you use in `sign-in-form.tsx` there

Reply:
Yeah i think i gonna create constant file in user module